server/database/index.ts

```diff
@@ -329,16 +332,39 @@ export const getFavoriteGenres = (userId: number) => {
   });
 };
 
+// Movies needs a get favorites
+export const getFavoriteMovies = (userId: number) => {
+  return User.findAll({
+    where: {userId: userId},
+    include: [
+      {
+        model: Movies,
+        through: {where: {userId: userId}}
+      }
+    ]
+  });
+};
+
 interface userObj {
   [key:string]: string;
 }
 
+export const getUserById = async (userId: number) => {
```

Comment:
database functions shouldn't have a response, that should be in the router.  This one could be implemented with `findByPk(userId);`, assuming that userId is the primary key for that user.

Reply:
Updated to be a cheeky little three liner, using `User.findByPk(userId)`. Thanks!